lineage/states/stateCommon.py

```diff
@@ -70,12 +70,14 @@ def gamma_estimator(gamma_obs, time_cen, gammas, x0):
     opt = {'gtol': 1e-12, 'ftol': 1e-12}
     bnd = (1.0, 100.0)
 
-    def GnLL(x, *args):
-        val, grad = GnLL_sep(x, *args)
-        return val, np.array(grad)
+    # def GnLL(x, *args):
+    #     val, grad = GnLL_sep(x, *args)
+    #     return val, np.array(grad)
 
-    res = minimize(GnLL, x0, jac=True, bounds=(bnd, bnd), args=arrgs, options=opt)
-    # assert (res.success is True) or ("maximum number of function evaluations is exceeded" in res.message)
+    res = minimize(GnLL_sep, x0, jac=True, bounds=(bnd, bnd), args=arrgs, method="trust-constr")
```

Comment:
Why does this change at all? Is it being used?

Reply:
Since the warning I mentioned in slack about line search failing, was also happening here, I replaced it with the `trust-constr` and was going to make the plots that use this function to make sure they give the same results and that our current figures are not wrong..

Reply:
Should I keep it or discard this change?

Reply:
If you aren't using this code for the new analysis anymore, it should be reverted.

Reply:
done.